Tests/Functional/ParamFetcherTest.php

```diff
@@ -33,20 +33,29 @@ public function setUp()
         $this->client = $this->createClient(['test_case' => 'ParamFetcher']);
     }
 
+    /**
+     * @group legacy
```

Comment:
Why are all the test here marked as legacy?

Reply:
They rely on using a scalar as a requirement, which is now deprecated. The @legacy annotation allows these tests to pass.